var/spack/repos/builtin/packages/py-archspec/package.py

```diff
@@ -12,11 +12,15 @@ class PyArchspec(PythonPackage):
 
     maintainers = ['alalazo']
 
+    version('0.1.3', sha256='a1aa7abde4d4ce38d115dfd572584906fa8e192e3272b8897e7b4fa1213ec27c')
+    version('0.1.2', sha256='8bb998370f0dc3e509d57c13724ab4109d761fd74af20da26fbe513b0fe01c46')
     version('0.1.1', sha256='34bafad493b41208857232e21776216d716de37ab051a6a4a1cc1653f7e26423')
+    version('0.1.0', sha256='a4431d0bbe9c9dd7b293c39d8e7590034d512ce5f5a1278a6cbdf61b33f7202d')
 
     depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
 
     depends_on('py-click@7.1.2:7', type=('build', 'run'))
     depends_on('py-six@1.13.0:1', type=('build', 'run'))
+    depends_on('py-poetry@1.1.12 ^python@3.7:', type=('build', 'run'))
```

Comment:
@haampie

Reply:
So, `poetry` should be needed only for development of `archspec`. The only required dependencies at runtime are `click` and `six` and since it's pure Python code it can just copied in the prefix :thinking:

Reply:
@alalazo The reason why I chose `^python@3.7:` is because, according to [Poetry's Github page](https://github.com/python-poetry/poetry), it only supports Python 3.7+. I'm not personally familiar with Poetry so if that is wrong, I can change that requirement. I was just basing it off what I could find. 

I'm not sure why it's failing to build without it though if it is only a development requirement. I have installed packages that depend on `archspec` before without issue, so I'm not sure what caused the change. Maybe a different Python version perhaps?

Reply:
I need to catch up with latest Python PEPs. It's failing because of [PEP517](https://www.python.org/dev/peps/pep-0517/). I am trying to see if I can work around that for the time being, since having a dependency on something that brings in `rust ` is not funny.

Reply:
That makes sense. If you agree, I can go ahead and remove the `depends_on('py-poetry@1.1.12 ^python@3.7:', type=('build', 'run'))`, but I'll leave the additional versions so those can be added in.